examples/assistant-ui/app/api/chat/route.ts

```diff
@@ -0,0 +1,17 @@
+import { openai } from '@ai-sdk/openai';
+import { getEdgeRuntimeResponse } from '@assistant-ui/react/edge';
+
+export const maxDuration = 30;
+
+export const POST = async (request: Request) => {
+  const requestData = await request.json();
+  console.log(requestData);
+
+  // return getExternalStoreRuntimeResponse({
+  //   // options: {
+  //   //   model: openai('gpt-4o'),
+  //   // },
+  //   requestData,
+  //   abortSignal: request.signal,
+  // });
```

Comment:
does this work? I want to check out Mastra with an existing assistant-ui app but saw this commented out and wondering if I'm going to have to figure out how to get this to work myself